src/client/entity-editor/name-section/name-section.js

```diff
@@ -113,7 +113,6 @@ class NameSection extends React.Component {
 	render() {
 		const {
 			disambiguationDefaultValue,
-			disambiguationVisible,
```

Comment:
There is a leftover `disambiguationVisible` in the JSDOC documentation comments on lines 58-59

Reply:
and sorry for the part , i forgot about JSDOC , my bad !